chat-commands.js

```diff
@@ -1268,7 +1269,7 @@ exports.commands = {
 			return this.errorReply("This command is only available in rooms");
 		}
 		if (!room.auth) {
-			this.sendReply("/roompromote - This room isn't designed for per-room moderation");
+			this.sendReply("/roompromote - This room isn't designed for per-room moderation.");
 			return this.sendReply("Before setting room staff, you need to set a room owner with /roomowner");
```

Comment:
why did you skip this?

Reply:
What do you mean?

Reply:
You added periods to a lot of other messages, but not this one.